x/staking/keeper/cons_pubkey.go

```diff
@@ -133,17 +131,9 @@ func (k Keeper) ValidatorIdentifier(ctx context.Context, newPk sdk.ConsAddress)
 
 // ExceedsMaxRotations returns true if the key rotations exceed the limit, currently we are limiting one rotation for unbonding period.
 func (k Keeper) ExceedsMaxRotations(ctx context.Context, valAddr sdk.ValAddress) error {
-	count := 0
-	rng := collections.NewPrefixedPairRange[[]byte, time.Time](valAddr)
-
-	if err := k.ValidatorConsensusKeyRotationRecordIndexKey.Walk(ctx, rng, func(key collections.Pair[[]byte, time.Time]) (stop bool, err error) {
-		count++
-		return count >= maxRotations, nil
-	}); err != nil {
-		return err
-	}
+	found, _ := k.ValidatorConsensusKeyRotationRecordIndexKey.Has(ctx, valAddr)
```

Comment:
I think the change makes sense to me since the rotation limit is 1. 

I would also rename the function to "IsAlreadyRotating"

Reply:
yes, ty